flang/lib/Lower/OpenACC.cpp

```diff
@@ -2681,18 +2691,18 @@ genACC(Fortran::lower::AbstractConverter &converter,
   if (blockDirective.v == llvm::acc::ACCD_parallel) {
     createComputeOp<mlir::acc::ParallelOp>(converter, currentLocation, eval,
                                            semanticsContext, stmtCtx,
-                                           accClauseList);
+                                           accClauseList, {});
```

Comment:
```suggestion
                                           accClauseList, /*combinedConstructs=*/{});
```
IF you want to keep it this way